EncoreTickets.SDK.Tests/UnitTests/Aws/AwsSqsTests.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Amazon.Extensions.NETCore.Setup;
+using Amazon.Runtime.CredentialManagement;
+using Amazon.SQS;
+using Amazon.SQS.Model;
+using EncoreTickets.SDK.Aws;
+using Moq;
+using NUnit.Framework;
+
+namespace EncoreTickets.SDK.Tests.UnitTests.Aws
+{
+    internal class AwsSqsTests : AwsSqs
+    {
+        public static AwsSqsMockers Mockers;
+
+        static AwsSqsTests()
+        {
+            Mockers = new AwsSqsMockers();
+        }
+
+        public AwsSqsTests() : base("profile_name", "region_name")
+        {
+        }
+
+        protected override ICredentialProfileStore CreateCredentialProfileStore()
+        {
+            return Mockers.SharedCredentialsFileMock.Object;
+        }
+
+        protected override IAmazonSQS CreateAmazonSqsClient(AWSOptions options)
+        {
+            return Mockers.ClientMock.Object;
+        }
+
+        [TestCase("dev", "eu-west-1")]
+        public void Constructor_IfProfileAndRegionArguments_InitializesClient(string profileName, string regionName)
+        {
+            AwsSqsWrapper.Mockers = new AwsSqsMockers();
+
+            var aws = new AwsSqsWrapper(profileName, regionName);
```

Comment:
Why do you test AwsSqsWrapper if it's not used anywhere outside these tests?

Reply:
AwsSqsWrapper was created specifically for testing the initialization of non-public members of AwsSqs